### DISCUSSION TITLE
Version Bumps

### DISCUSSION THREAD

**Discussion by waldronmatt** [ghiscoding/lerna-lite#807] [Q&A] [answered]
Hello,

I would like your thoughts on a feature for Lerna to ignore changes made to package dependencies installed via workspace like pnpm workspace*. I have an eslint config in my monorepo that is consumed by all my packages. Every time eslint updates, this triggers the version for package only text. This isn't necessary to have versions update all the time when worksace dependencies will usually get the most recent changes in the monorepo. What I will probably do moving forward is to configure my renovate to not use conventional commits so that Lerna Lite --skip-bump-only-releases works correctly to suppress releases. But it would be nice to have changes ignored entirely including the changelog for workspace dependencies. I have also looked through the original Lerna repo issues and this library cli --skip-bump-only-releases seems to be the best option right now. Would love to hear your thoughts on this.

**Comment by ghiscoding** [comment, marked as answer]
The `--skip-bump-only-releases` is an option that I added in this project only (it doesn't exist in the original Lerna) and the only thing it does is to skip the creation of GitHub releases and it is only useful with the `independent` mode, it also has no effect whatsoever on the version bump (they will still be bumped). I also use `workspace:*` in this project and eslint dependencies also get updated often (I found recently that they have automated Monday releases), but even then they are typically updated under a `chore` tag which isn't supposed to bump the version but anyway the bump is not calculated by Lerna/Lerna-Lite, it's calculated by the `conventional-changelog-recommend-bump` dependency. Anyway, I don't have time neither the knowledge to add complex options, my main goal with this project is to keep up to date and I'm also open for any contributions.

**Reply by waldronmatt** [reply]
Thanks for your response. I'll look to how lerna-lite is set up for inspiration. Even though many of my prs are merged with chore in the pr title lerna does seem to increment, but that seems to be expected behavior as you explained above and findings from open issues on the lerna repo. Agreed it seems like this request is beyond the scope of lerna. I appreciate your work on this project.
